fastestimator/op/tensorop/augmentation/mixup_batch.py

```diff
@@ -50,30 +50,49 @@ def __init__(self,
                  mode: Union[None, str, Iterable[str]] = 'train',
                  ds_id: Union[None, str, Iterable[str]] = None,
                  alpha: float = 1.0,
-                 shared_beta: bool = True):
+                 shared_beta: bool = False):
         assert alpha > 0, "MixUp alpha value must be greater than zero"
         super().__init__(inputs=inputs, outputs=outputs, mode=mode, ds_id=ds_id)
-        assert len(self.outputs) == len(self.inputs) + 1, "MixUpBatch requires 1 more output than inputs"
         self.alpha = alpha
         self.beta = None
         self.shared_beta = shared_beta
-        self.in_list, self.out_list = True, True
+        #self.in_list, self.out_list = True, True
```

Comment:
if you're dropping multi-io you can delete the line rather than leaving commented out